docs/source/using_yosys/more_scripting/data_flow_tracking.rst

```diff
@@ -0,0 +1,71 @@
+Dataflow tracking
+-------------------
+Yosys can be used to answer questions such as "can this signal affect this other signal?" via its *dataflow tracking* support.
+For this, four special cells, ``$get_tag``, ``$set_tag``, ``$overwrite_tag`` and ``$original_tag`` are inserted into the design and then the ``dft_tag`` is run, which converts these cells into ordinary logic.
```

Comment:
Cells and commands will automatically link to the corresponding documentation if they are put in single back tick quotes (while still being code formatted).
```suggestion
For this, four special cells, `$get_tag`, `$set_tag`, `$overwrite_tag` and `$original_tag` are inserted into the design and then the `dft_tag` is run, which converts these cells into ordinary logic.
```
Most of the documentation does this for all references, but sometimes that can be overkill if everything is a hyperlink.  I would at least follow the acronym style of having a link on the first reference, the rest are at your discretion.